plugins/signals/src/api/SignalClient.ts

```diff
@@ -135,7 +135,7 @@ export class SignalClient implements SignalApi {
   }
 
   private async connect() {
-    if (this.ws && this.ws.readyState === WebSocket.OPEN) {
```

Comment:
Hm. But what about CLOSING and CLOSED? Are those not of concern, even if because of a rollout going on or flaky network or whatnot? I'm not an expert on ws but will active reconnects _never_ be a thing to think about?

Reply:
The state is handled below when the instance is first created and it will reconnect when the session is closed. Issue is that now it creates multiple Websocket instances overwriting the `this.ws` and messing up also the state handling

Reply:
Hm, maybe we could change this to `readyState !== WebSocket.CLOSED` so it should handle other parts properly